storybook/serve.json

```diff
@@ -0,0 +1,6 @@
+{
+  "public": "./storybook-static",
+  "cleanUrls": false,
+  "directoryListing": false,
+  "redirects": [{ "source": "/", "destination": "/index.html" }]
+}
```

Comment:
Maybe not obvious either, this is the config file for the `serve` command in the package json. It needed a bit of configuration to be able to serve the production-build of storybook in a comparable fashion to what is currently live.